DesignSafe-CI/portal
# removed Jupyter link from Community Data and Published

## Overview: ##
Removed Jupyter link from "Published" and "Community Data" 

## PR Status: ##


## Related Jira tickets: ##

* [DES-2281](https://jira.tacc.utexas.edu/browse/DES-2281)

## Summary of Changes: ##
In the file-listing template, I commented out the section that put the Jupyter link in the table.

## Testing Steps: ##
1. Community Data --> Jupyter Notebooks. Notice no "Jupyter" links in the "Open with" column for files that end with pynb.
2. Published --> keyword search: Jupyter. Notice no "Jupyter" links in the "Open with" column for files that end with pynb.

## UI Photos:
![Screen Shot 2022-07-19 at 10 52 03 AM](https://user-images.githubusercontent.com/35277477/179997154-681d4bf6-a33e-4461-9633-38df887663f8.png)


## Notes: ##
